src/Models/Meetup.php

```diff
@@ -44,7 +45,7 @@ class Meetup extends Model
 
     public function getLogoPath(): string
     {
-        return asset("storage/" . $this->attributes["logo_path"]);
+        return asset($this->attributes["logo_path"] ? "storage/" . $this->attributes["logo_path"] : Constants::MEETUP_DEFAULT_LOGO_PATH);
```

Comment:
Maybe this would be helpful? https://laravel.com/docs/9.x/eloquent#default-attribute-values